harambe/parser/parser.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, schema: Schema, data: ScrapeResult, message: str):
         )
 
 
+class RequiredFieldsError(Exception):
```

Comment:
Maybe make this extend SchemaValidationError? Thoughts @awtkns

Reply:
Yeah lets make this a schema validation error